resources/views/themes/default1/front/clients/show-order.blade.php

```diff
@@ -316,6 +316,13 @@
          <link rel="stylesheet" type="text/css" href="//cdn.datatables.net/1.10.12/css/jquery.dataTables.min.css" />
         <script src="//cdn.datatables.net/1.10.12/js/jquery.dataTables.min.js"></script>
         <script type="text/javascript">
+              $(document).ready(function(){
+                let ele = document.getElementById("s_key").textContent;
+                 let finalele = ele.match(/.{1,4}/g).join('-');
```

Comment:
what does this do?

Reply:
Puts '-' after every 4 characters of license code